src/state/store.js

```diff
@@ -111,16 +103,10 @@ const initialState = {
 
 export const store = createStore(initialState);
 
-store.setProbe = (name) => {
-  // get matching probe here
-  const probe = get(probeSet).find((d) => d.name === name);
-  store.setField('probe', probe);
-};
-
 store.reset = () => {
   store.reinitialize({
     auth: store.getState().auth,
-    probe: { name: undefined },
+    probeName: undefined,
```

Comment:
I like uninitialized strings to be the empty string `''` so it's still set to the same type.

Reply:
Good call! Yes, this would match the above. Thank you!